### DISCUSSION TITLE
How do I download a publicly shared file?

### DISCUSSION THREAD

**Discussion by NightMachinery** [abraunegg/onedrive#2175] [Q&A] [answered]
How do I download a publicly shared file (e.g., [this](https://onedrive.live.com/?authkey=%21AOaPhONxGCKIlrg&cid=910FC0C1DD8E9B39&id=910FC0C1DD8E9B39%21163&parId=910FC0C1DD8E9B39%21104&o=OneUp))? Preferably without logging in.

**Comment by abraunegg** [comment, marked as answer]
@NightMachinery 
> How do I download a publicly shared file (e.g., [this](https://onedrive.live.com/?authkey=%21AOaPhONxGCKIlrg&cid=910FC0C1DD8E9B39&id=910FC0C1DD8E9B39%21163&parId=910FC0C1DD8E9B39%21104&o=OneUp))? Preferably without logging in.


Not with this client - a web browser maybe ?

This client requires you to authenticate and login.

**Reply by NightMachinery** [reply]
Suppose I do authenticate. How do I download the link then?

**Reply by abraunegg** [reply]
Using your web browser, add the link to your OneDrive account, and access that way - this is most likely the only solution for you.
